eng/common/scripts/Package-Properties.ps1

```diff
@@ -110,7 +99,7 @@ function Get-PkgProperties
 # Takes ServiceName and Repo Root Directory
 # Returns important properties for each package in the specified service, or entire repo if the serviceName is not specified
 # Returns an Table of service key to array values of PS Object with properties @ { pkgName, pkgVersion, pkgDirectoryPath, pkgReadMePath, pkgChangeLogPath }
-function Get-AllPkgProperties ([string]$ServiceDirectory = $null)
+function Get-AllPkgProperties ([string]$ServiceDirectory = $null, [bool]$includeMgmt = $true)
```

Comment:
Should we just take a filter option here? Something like a default of `ci*.yml`?

Reply:
The order we read from ci.yml is (ci.data.yml, ci.mgmt.yml, ci.yml) which we do not list the most important one on the very top of the page. That's why I specify the ci file order (ci.yml, ci.data.yml, ci.mgmt.yml) also check the file existence as well in helper function.

Reply:
Have you seen what the order is when it comes back? I personally don't know that it is worth the extra complexity. We don't need to worry about an exists check when you use a wildcard as it will only process the files that it finds.

Reply:
It is in the order of ci.data.yml, ci.mgmt.yml, ci.yml if we use ci.*yml